blocks/accordion/accordion.css

```diff
@@ -0,0 +1,15 @@
+.accordion {
+  border: none;
+  padding: 1px;
+}
+
+.accordion-title {
+  background-color: darkgrey;
+  color: white;
+  font-size: 22px !important;
```

Comment:
<img width="223" alt="Screen Shot 2022-12-05 at 2 35 51 PM" src="https://user-images.githubusercontent.com/6012307/205650348-e91841f7-6b83-4a8d-8dee-b66f8e3da3bc.png">

I used `!important` because I don't know where these default styles are coming from ?